workflows/metatdenovo.nf

```diff
@@ -71,6 +72,17 @@ include { DIGINORM } from '../subworkflows/local/diginorm' addParams(
     diginorm_extractpairedreads_options: diginorm_extractpairedreads_options
 )
 
+//
+// SUBWORKFLOW: Consisting of nf-core/modules
+//
+def prokka_options              = modules['prokka']
+def cat_options                 = modules['cat_cat']
```

Comment:
Could be one each for `gff_cat` etc. Maybe better to just rename the `modules.config` entry to `prokka_cat`?